hack/cluster/deploy-cluster.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+az login
+echo "Put name of resource to be created"
+read resrc
+echo "Enter location"
+read location
```

Comment:
Could you replace those variables with environment variables? So that it could be set easily from Makefile